felix/bpf/bpf.go

```
@@ -387,6 +387,69 @@ type mapEntry struct {
 	Err   string   `json:"error"`
 }
 
+func (me *mapEntry) UnmarshalJSON(data []byte) error {
+	type entry struct {
+		Key   []string    `json:"key"`
+		Value interface{} `json:"value"`
+		Err   string      `json:"error"`
+	}
+
+	if string(data) == "null" {
+		return nil
+	}
+
+	var e entry
+	err := json.Unmarshal(data, &e)
+	if err != nil {
+		return nil
+	}
+
+	v, ok := e.Value.([]interface{})
+	if !ok {
+
+		return nil
+	}
+
+	hexbytes := make([]string, len(v))
+	for i, x := range v {
+		hexbytes[i] = x.(string)
+	}
+
+	*me = mapEntry{
+		Key:   e.Key,
+		Value: hexbytes,
+		Err:   e.Err,
+	}
+
+	return nil
+}
+
+type hexMap []mapEntry
+
+func (hm *hexMap) UnmarshalJSON(data []byte) error {
+	if string(data) == "null" {
+		return nil
+	}
+
+	var m []mapEntry
+	err := json.Unmarshal(data, &m)
+	if err != nil {
+		return nil
```

Why suppress the error?